studio/schemas/objects/jobPosting.ts

```diff
@@ -0,0 +1,53 @@
+import { defineType } from "sanity";
+
+import { isInternationalizedString } from "studio/lib/interfaces/global";
+import { firstTranslation } from "studio/utils/i18n";
+
+export const jobPostingID = "jobPosting";
+
+const jobPosting = defineType({
+  name: jobPostingID,
+  title: "Job Posting",
+  type: "object",
+  fields: [
+    {
+      title: "Role",
+      name: "role",
+      type: "internationalizedArrayString",
+      description: "The name of the role",
+    },
+    {
+      title: "Location",
```

Comment:
You might have already discussed this with either the Ole Petter or Marius, if so please ignore this comment! At the current website we have the option of listing a job to multiple locations if the listing is not location-specific, eg. "Fagleder tjenestedesign", should we also do that here?

Reply:
Hmm, I guess we could make the field optional, in which case no location means that it's not tied to a specific location 🤔

Reply:
I would send a DM to Marius Krakeli, who is responsible for recruitment in Variant, I believe he has some opinions on whether it should be optional or a list of locations ☺️

Reply:
> I would send a DM to Marius Krakeli, who is responsible for recruitment in Variant, I believe he has some opinions on whether it should be optional or a list of locations ☺️

Done! I'll keep you posted

Reply:
<img width="1269" alt="Screenshot 2024-11-26 at 10 36 23" src="https://github.com/user-attachments/assets/2d2dd5ac-cd9a-4b02-a906-2f3f624b77b1">